CONTRIBUTING.md

```diff
@@ -7,3 +7,31 @@ Pull requests will be evaluated by the repository guardians on a schedule and if
 
 All contributors retain the original copyright to their stuff, but by contributing to this project, you grant a world-wide, royalty-free, perpetual, irrevocable, non-exclusive, transferable license to all users **under the terms of the license under which this project is distributed.**
 
+## Git Workflow
+
+### Creating a Pull Request
+
+- Branch new features off the `develop` branch.
+- Naming convention based off the problem at hand.  (i.e. update-export-for-fault-tolerance )
+- Before pushing your new feature up to prod make sure to pull in the latest develop and fix any merge conflicts that might happen.
+`git pull origin develop`
+- Push your branch up to github.
+`git push origin <branchname>`
+- open a pull request against develop and request a team member review it.
+
+
+### Merging your code into develop.
+
+- GitHub will not let you merge code that has not passed the following checks
+  - The pipeline builds correctly (all unit tests pass)
+
+  - Another developer has performed code review and you have implemented requested changes
+
+  - Your branch is not behind develop in terms of commits.  If it is, pull the changes into your branch and resolve any merge conflicts.
```

Comment:
Technically, your branch can be behind develop any number of commits, so long as none of them are conflicting.